ofrak_core/ofrak/cli/command/gui.py

```diff
@@ -31,9 +31,17 @@ def create_parser(self, ofrak_subparsers):
             help="Set GUI server host port.",
             default=8080,
         )
+        gui_parser.add_argument(
+            "--no-browser",
+            action="store_true",
+            help="Don't open the browser to the OFRAK GUI",
+        )
         self.add_ofrak_arguments(gui_parser)
         return gui_parser
 
     async def ofrak_func(self, ofrak_context: OFRAKContext, args: Namespace):  # pragma: no cover
-        server = await open_gui(args.hostname, args.port)
+        if not args.no_browser:
+            server = await open_gui(args.hostname, args.port)
```

Comment:
I'd rather there be one consistent function we call to launch the GUI, and it should probably be `open_gui`. That function could just have an option to try to open it in the browser or not. I also think the print about where the GUI is being served should print regardless of whether the browser is opened or not.